reana_server/reana_admin.py

```diff
@@ -584,16 +610,18 @@ def set_quota_limit(ctx, admin_access_token, emails, resource_name, limit):
     "quota-set-default-limits",
     help="Set default quota limits to users that do not have any.",
 )
-@admin_access_token_option
 @click.pass_context
-def set_default_quota_limit(ctx, admin_access_token):
+def set_default_quota_limit(ctx):
     """Set default quota limits to users that do not have any."""
     users_without_quota_limits = User.query.filter(~User.resources.any()).all()
+    if not users_without_quota_limits:
+        click.secho("There are no users without quota limits.", fg="green")
+        sys.exit(0)
     for resource in Resource.query:
         ctx.invoke(
             set_quota_limit,
-            admin_access_token=admin_access_token,
             emails=[user.email for user in users_without_quota_limits],
             resource_name=resource.name,
+            resource_type=resource.type_.name,
```

Comment:
This can be omitted as the code, as it is now, will always check `resource_name` first and ignore `resource_type` but we can keep it just in case.